security_monkey/alerters/test_slack_message.py

```diff
@@ -0,0 +1,18 @@
+#Just a quick program to test your communication of your slack bot and security monkey
```

Comment:
For this, please consider mocking the slack client. I do something similar in HubCommander: 
https://github.com/Netflix/hubcommander/blob/develop/tests/conftest.py#L77-L83

I basically mock out the Slack Client, and then when it's used:
https://github.com/Netflix/hubcommander/blob/develop/tests/test_slack_comm.py#L14-L16 
^^ To verify what was passed in was correct.